cmd/docs/README.md

```diff
@@ -0,0 +1,97 @@
+# WSO2 Micro Integrator Command Line tool (MI-CLI)
+
+View and manage the micro integrator runtime using the “mi” command line tool. Some of the usages of the command line tool include,
+1. Get a list of deployment runtime artifacts
+2. Inspect details of each runtime artifact such as a proxy service or an API.
+3. Get the invocation endpoint of an artifact
+
+### Running
+
+Extract the compressed archive generated to the desired location.
+
+Add the MI CLI bin folder to PATH in unix-based Operating System (Linux, Solaris and Mac OS X)
+
+`export PATH=/path/to/mi/cli/directory/bin:$PATH`
+
+Then execute, 
+
+`$ mi`
+
+Execute mi --help for further instructions.
+
+### Add Command Autocompletion
+
+`$ source /path/to/mi/cli/directory/mi_bash_completion.sh`
```

Comment:
Shall we also mention that this is only available for UNIX based operating systems?